File: week1/day4/labs/triangle.rb
puts 'How Big Would you like your Triangle'
triangle_size = gets.to_i

rows = 1
characters = 0

while characters < triangle_size
  spacing = triangle_size - rows
  spacing.times { print " "}
  rows.times { print " O" }
  puts ""
  rows += 1
  characters += 1
end



# *********************JAVASCRIPT*******************************

# const readline = require('readline');
# const rl = readline.createInterface({
#   input: process.stdin,
#   output: process.stdout
# });
#
# rl.question('How Tall Would You Like Your Pyramid to be? (0-100)? ', function (number1) {
#   let size = parseInt(number1);
#   for (i = 1; i <= size; i += 1) {
#     console.log( " ".repeat(size - i) + " #".repeat(i) );
#   }
#   rl.close();
# });

File: week1/day2/labs/triangle.js
// Write a program that prompts the user for the length of each side of a triangle then prints out the area of the triangle and circumference of a the triangle

const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Please Enter the Height of a Triangle: ', function (side1) {
  rl.question('Please Enter the Base of the Triangle: ', function (side2) {
    rl.question('Please Enter the Final Side of the Triangle: ', function (side3) {
      sideA = parseInt(side1)
      sideB = parseInt(side2)
      sideC = parseInt(side3)
      let circumference = `${sideA} + ${sideB} + ${sideC}`
      console.log(`The Circumference of your Triangle is ${circumference}`);
      let Area = Math.sqrt(`${circumference} * ( (${circumference} - ${sideA})*(${circumference} - ${sideB})*(${circumference} - ${sideC}) )`)
      console.log(`The Area of your Triangle is ${Area} `);
      rl.close();
    });
  });
});
